conf/config.go

```diff
@@ -90,7 +90,8 @@ type MenderConfig struct {
 }
 
 type DBusConfig struct {
-	Enabled bool
+	Enabled                               bool
+	UpdateControlMapExpirationTimeSeconds int
```

Comment:
It should be outside the `DBusConfig` structure. It is not DBus specific, DBus is just one of the ways that it can be updated.